.gitlab-ci.yml

```diff
@@ -13,3 +13,28 @@ include:
       - '.gitlab-ci-check-license.yml'
       - '.gitlab-ci-check-docker-build.yml'
       - '.gitlab-ci-github-status-updates.yml'
+
+.trigger:update-dependencies:mender-docs-site:
+  inherit:
+    variables: false
```

Comment:
Nice!

Reply:
FYI, this was a big gotcha during a release I coordinated some time ago. See https://tracker.mender.io/browse/QA-352